api/scripts/database/drop-database.js

```diff
@@ -34,6 +34,8 @@ PgClient.getClient(url.href).then(async (client) => {
   } catch (error) {
     if (error.code === PGSQL_NON_EXISTENT_DATABASE_ERROR) {
       logger.info(`Database ${DB_TO_DELETE_NAME} does not exist`);
+    } else {
+      logger.error(`Could not be done because ${error.detail}`);
```

Comment:
```suggestion
      logger.error(`Database drop could not be done because ${error.detail}`);
```

```
      logger.error(`Database drop could not be done: ${error.detail}`);
```

Reply:
Done 👍